kitty/css/style.css

```diff
@@ -0,0 +1,530 @@
+  content: "\e90e";
+  }
+  .icon-disable:before {
+  content: "\e90f";
+  }
+  .icon-pointer-down:before {
+  content: "\e914";
+  }
+  .icon-pointer-up:before {
+  content: "\e916";
+  }
+  .icon-pointer-left:before {
+  content: "\e917";
+  }
+  .icon-pointer-right:before {
+  content: "\e918";
+  }
+  .icon-csv:before {
+  content: "\e919";
+  }
+  .icon-zip:before {
+  content: "\e91a";
+  }
+  .icon-attention-sym:before {
+  content: "\e91c";
+  }
+  .icon-question-sym:before {
+  content: "\e91d";
+  }
+  .icon-star-solid:before {
+  content: "\e91e";
+  }
+  .icon-pause:before {
+  content: "\e91b";
+  }
+  .icon-openbox:before {
+  content: "\e91f";
+  }
+  .icon-resume:before {
+  content: "\e920";
+  }
+
+
+body{
+    background-color: #f1f1f4;
+    font-size:1em;
+}
+
+header .row{
+  margin:0;
+  padding:0;
+  background-color: #03BA5B;
+  padding:10px 0 0 20px;;
+  height:50px;
+  width:100%;
+  position:fixed;
+  z-index: 10;
+  box-shadow: 0px 0px 5px #414141;
+  font-size:12px;
+}
+
+
+.row{
+  margin:0px 0 0 200px;
+  padding-top:74px;
+  
+}
+
+.col h1{
+  font-size:32px;
+  line-height:32px;
+    text-transform: capitalize;
+
+}
+
+.col p{
+ font-size:12px;
+
+}
+
+.card{
+  margin-bottom:12px;
+}
+
+
+/* Navigation */
+nav{
+    background-color: #4B555B;
+    color:white;
+    text-transform: uppercase;
+    font-size:12px;
+    width:200px;
+    min-height: 100vh;
+    float:left;
+    margin-top:50px;
+    position:fixed;
+    z-index:9;
+}
+
+nav span{
+    padding:10px;
+    font-size:16px;
+  
+}
+nav a{
+    color:white;
+    text-decoration: none;
+}
+
+nav li:hover{
+    background-color:#343B3F;
+}
+
+nav li{
+   padding:20px;
+   
+ 
+}
+
+/* Top Nav */
+
+.profile{
+margin-top:-10px;
+margin-right:20px;
+float:right;
+height:50px;
+padding:19px 0 19px 0;
+font-size:12px;
+color:white;
+}
+
+.profile-comment{
+    margin-top:-10px;
+    margin-right:20px;
+    float:left;
+    height:50px;
+    padding:19px 0 19px 0;
+    font-size:12px;
+    color:white;
+    }
+
+.profile-photo{
+ 
+    margin:-19px 0 0 0px; 
+    }
+
+.clip-circle{
+    clip-path: circle(15px at center);
+}
+
+
+/* Body */
+
+.box{
+    border:#E3E3E3 .5px solid;
+    background-color:white;
+    padding:20px;
+    width:100%;
+    border-radius: 3px;
+    margin-bottom:10px;
+}
+
+.container-fluid{
+padding:0;
+margin:0;
+}
+
+
+.search span{
+    padding:29px 0 29px 30px;
+    height:70px;
+}
+
+.search{
+    background-color:white;
+    margin-bottom: 12px;
+    border-radius: 4px;
+    border:#E3E3E3 1px solid;
+}
+
+input{
+    width:90%;
+    height:70px;
+    padding-left:10px;
+    border:none;
+    font-size:16px;
+}
+
+.recent-activities{
+    float:left;
+    height:auto;
+}
+
+.pending-items{
+    border-radius: 4px;
+    height:80px;
+    margin-top:24px;
+    padding:33px;
+}
+
+.pending-items h2{
+    float:left;
+    font-size:14px;
+}
+
+.pending-items p{
+    float:right;
+    line-height:12px;
+    Color:rgb(38, 188, 185);
```

Comment:
Color doesn't need the capitalization, and try to keep the colour definitions to one type (since you're using HEX everywhere else)!